kong/clustering/services/sync/rpc.lua

```diff
@@ -389,7 +390,7 @@ function sync_once_impl(premature, retry_count)
     return
   end
 
-  sync_handler()
+  local _, err = sync_handler()
```

Comment:
I think that `sync_handler()` won't return anything? could you confirm it?

Reply:
fixed